README.md

```diff
@@ -21,12 +20,128 @@ Or install it yourself as:
 $ gem install wor-paginate
 ```
 
+Then you can run `rails generate wor:paginate:install` to create the initializer for configuration details, including default formatter class, page and limit params and default page number.
```

Comment:
I don't get what should be understood with `page and limit params and default page number`